solr/solrj/src/test/org/apache/solr/client/solrj/response/TestSpellCheckResponse.java

```diff
@@ -27,21 +29,22 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 
-import java.util.List;
+import junit.framework.Assert;
 
 /**
  * Test for SpellCheckComponent's response in Solrj
  *
  *
  * @since solr 1.3
  */
-public class TestSpellCheckResponse extends SolrJettyTestBase {
+public class TestSpellCheckResponse extends EmbeddedSolrServerTestBase {
```

Comment:
[Q] Why are you changing the base class of this test here?  I'm fine with introducing a new base test-class in this PR, and maybe we want to eventually change other classes to use this new base.  But I'd rather that happen in a separate PR.

(That way, if tests start failing it's clear what's at fault.  With the changes bundled together like this it's harder to tell whether the bugfixes caused any hypothetical test failures, or the test-changes themselves are unsound.)

Reply:
Because the test uses the hidden EmbeddedSolrServer, so moved it to the new base